scripts/barclamp_install.rb

```diff
@@ -54,10 +55,12 @@ def usage()
 
 usage if ARGV.length < 1
 
-tmpdir = "/tmp/bc_install-#{Process.pid}-#{Kernel.rand(65535)}"
+tmpdir = Dir.mktmpdir "component-install-"
```

Comment:
Apologies for my earlier comment, but I suddenly realize that we don't use `tmpdir` at all? So maybe we should just drop all mentions of it?